spec/features/responses/response_spec.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+feature "responses", js: true do
```

Comment:
Ah, yes done. That directory name confuses me, should it be `responses` instead of `results`? There are a bunch of specs in there specifically for responses.

Reply:
Yeah we should probably do that. I was at one time scheming on namespacing stuff in NEMO between form design and response handling, but it didn't really go anywhere...